managed/CounterStrikeSharp.API/Modules/Menu/BaseMenu.cs

```diff
@@ -15,48 +15,35 @@
  */
 
 using System.Collections.Generic;
+using CounterStrikeSharp.API.Modules.Utils;
 
 namespace CounterStrikeSharp.API.Modules.Menu
 {
-    public abstract class BaseMenu : IMenu
+    public class BaseMenu : IMenu
     {
         public string Title { get; set; }
         public List<ChatMenuOption> MenuOptions { get; } = new();
-     
+
         protected BaseMenu(string title)
         {
             Title = title;
         }
-        
-        public virtual ChatMenuOption AddMenuOption(string display, Action<CCSPlayerController, ChatMenuOption> onSelect, bool disabled = false)
+
+        public ChatMenuOption AddMenuOption(string display, Action<CCSPlayerController, ChatMenuOption> onSelect,
+            bool disabled = false)
```

Comment:
Try and avoid formatting changes as they make it harder to figure out what your actual changes are